tests/s3/test_multipart_abort_copy.py

```diff
@@ -33,13 +33,15 @@
 from commons.utils.s3_utils import get_unaligned_parts
 from commons.utils.s3_utils import get_precalculated_parts
 from commons.utils.s3_utils import get_multipart_etag
+from commons.utils.s3_utils import assert_s3_err_msg
 from commons.utils.system_utils import create_file, remove_file, path_exists
 from commons.utils.system_utils import backup_or_restore_files, make_dirs, remove_dirs
 from commons.utils import assert_utils
 from commons.params import TEST_DATA_FOLDER
+from commons import constants as const
 from config.s3 import S3_CFG
 from config.s3 import MPART_CFG
-from libs.s3 import S3H_OBJ
+from libs.s3 import S3H_OBJ, CMN_CFG
```

Comment:
Imports should be on independent lines as per the convention we have decided to follow. Can you also refactor other  import statements in this file to follow that approach?